app/overrides/layouts/decidim/mailer/add_css.html.erb.deface

```diff
@@ -0,0 +1,15 @@
+<!-- insert_after "erb[loud]:contains('stylesheet_pack_tag')" -->
+
+<style>
+a {
+    color: red;
```

Comment:
```suggestion
    color: #f22e41;
```

Reply:
I think this is the right color according to https://github.com/user-attachments/files/19009943/disseny.butlleti.coopmmunity-1.pdf

Also there's a part with a serif font instead of arial-sans
![image](https://github.com/user-attachments/assets/d4c18b2b-1e97-4eae-b304-ad5f81cb3d5a)